package.json

```diff
@@ -83,7 +83,7 @@
     "postcss-browser-reporter": "^0.5.0",
     "postcss-loader": "^2.0.6",
     "postcss-reporter": "^5.0.0",
-    "prettier": "1.9.2",
+    "prettier": "^1.9.2",
```

Comment:
Especially with formatters, I'd fix the version. Should we bump it here manually to `1.16.4` ?

Reply:
You are right, we can fix it here. Will limit the amount of confusion related to prettier changes. Updated in db6f84d61423f6cb9e0d02782a17a225450be2b8